Makefile

```diff
@@ -0,0 +1,16 @@
+# Build website and output to _site folder
+.PHONY: release
+release:
+	rm -rf _site
+	docker buildx bake release
+
+# Vendor Gemfile.lock
+.PHONY: vendor
+vendor:
+	docker buildx bake vendor
+
+# Deploy website and run it through Docker compose
+# Available in your browser at http://localhost:4000
+.PHONY: deploy
+deploy:
```

Comment:
Bike shed!

- I found `deploy` as a name a bit confusing; it sounds scary (will this deploy to "production"?). Alternatives? Perhaps `preview`? Other alternatives?
- `make` uses the first target as default; perhaps `deploy` should be first in the list, so that just typing `make` will build and run the preview (instead of `release`,  which should be more of an "conscious decision")
- alternatively, we could add the `make help` snippet we have in other repositories, and show `make help` as default

Reply:
I might remove nginx and directly use `jekyll serve` in a follow-up so we can use the `watch` feature.